pkg/controller/client.go

```diff
@@ -73,12 +72,12 @@ func (c *XdsClient) createStreamClient() error {
 
 	c.client = service_discovery_v3.NewAggregatedDiscoveryServiceClient(c.grpcConn)
 
-	if bpfConfig.EnableKmesh {
+	if bpfConfig.AdsEnabled() {
 		if err = c.AdsStream.AdsStreamCreateAndSend(c.client, c.ctx); err != nil {
 			_ = c.grpcConn.Close()
 			return fmt.Errorf("create ads stream failed, %s", err)
 		}
-	} else if bpfConfig.EnableKmeshWorkload {
+	} else if bpfConfig.WdsEnabled() {
```

Comment:
If the default is to use the mode of workload, is it more efficient to first determine `bpfconfig.WdsEnable` or not.